apps/compiler/includes/functions.sh

```diff
@@ -1,16 +1,11 @@
 
 function comp_clean() {
-  echo "Cleaning build files"
+  DIRTOCLEAN=${$BUILDPATH:-"var/build/obj"}
```

Comment:
```suggestion
  DIRTOCLEAN=${$BUILDPATH:-'var/build/obj'}
```

Reply:
think this should be single quotes instead, so you don't get the bad substitution error

Reply:
actually removing the second $ fixes this:
![image](https://user-images.githubusercontent.com/4015312/116009190-feb33180-a5cc-11eb-9c0b-5e8b9a2570be.png)